action.yaml

```diff
@@ -40,6 +44,9 @@ inputs:
   codedeploy_group:
     description: 'AWS CodeDeploy Application Group'
     required: true
+  codedeploy_config_name:
+    description: 'Override the AWS CodeDeploy configuration name'
+    required: false
```

Comment:
Not following this change - you should make this a separate pull request so we don't mix the bundle type issue with this.